odoo/addons/base/models/ir_model.py

```diff
@@ -17,7 +17,7 @@
 from odoo import api, fields, models, tools, _, _lt, Command
 from odoo.exceptions import AccessError, UserError, ValidationError
 from odoo.osv import expression
-from odoo.tools import format_list, pycompat, unique, OrderedSet
+from odoo.tools import format_list, pycompat, unique, OrderedSet, sql as sqltools
```

Comment:
Can you leave a TODO note so that we use `SQL` wrapper from tools and to remove direct psycopg2 import (if possible). Or alias `from psycopg2 import pg_sql`.